src/stylesheets/pages/_home.scss

```diff
@@ -1,3 +1,8 @@
 // -----------------------------------------------------------------------------
 // This file contains styles that are specific to the home page.
 // -----------------------------------------------------------------------------
+.main {
+  color: blue;
```

Comment:
It may break a lot of stuff. And what's the reason of this code?

Reply:
I have no idea where this code comes from